config/routes.rb

```diff
@@ -3,4 +3,12 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/chefs/:id", to: "chefs#show"
+  get "/chefs/:chef_id/ingredients", to: "chefs/ingredients#index"
+
+  get "/dishes/:id", to: "dishes#show"
+
+  post "/dishes/:dishes_id/ingredients", to: "dish_ingredients#create"
+  delete "/dishes/:dishes_id/ingredients/delete", to: "dish_ingredients#destroy"
```

Comment:
A more conventional path here would be "/dishes/:dish_id/ingredients/:id" instead of /delete

Because you can pass the ingredient id in the path, that removes the need to pass it as a parameter in your button! Just a bit easier (but nice work figuring out that parameter pass in your view)